virtual_ecosystem/models/litter/env_factors.py

```diff
@@ -69,3 +144,33 @@ def calculate_soil_water_effect_on_litter_decomp(
     ) ** moisture_response_curvature
 
     return 1 - supression
+
+
+def average_over_microbially_active_layers(
+    environmental_variable: DataArray, layer_structure: LayerStructure
+) -> NDArray[np.float32]:
+    """Average an environmental variable over the microbially active layers.
+
+    This average is weighted by how much of the microbially active depth lies within
+    each layer.
+
+    Args:
+        environmental_variable: The environmental variable to be averaged
+        layer_structure: The LayerStructure instance for the simulation.
+
+    Returns:
+        The average of the environmental variable across the soil depth considered to be
+        microbially active.
+    """
+
```

Comment:
I think for the below ground temperature it would make sense to include the surface temperature (which we might want to move closer to the surface, maybe only 2 cm or so) when calculating where you sit on the temperature profile, else you will not have a gradient but just the value of the topsoil layer of that is deeper than you microbial activity. Am I understanding this correctly?

Reply:
It is at the actual depth which is why we do the vertical averaging I thought

Reply:
Oh right! Yes then I absolutely need to include the surface temperature in the calculation (and rework the calculation)

Reply:
I've created an issue to address this #535